src/components/ArticlePage.jsx

```diff
@@ -35,6 +55,7 @@ function ArticlePage() {
             <h3 style={{paddingTop:20}}>Written by: {article.author}</h3>
             <p style={{paddingTop: 20}}>{article.body}</p>
 
+            <ThumbUpRoundedIcon color={vote ? "primary" : ""} onClick={incrementVote}/> {article.votes + voteCount}  <ThumbDownRoundedIcon color={downVote ? "error" : ""} onClick={decrementVote} />
```

Comment:
You should avoid changing/manipulating your vote amount like this. I would suggest having [votes, setVotes] in state and invoking setVotes in your increase and decrease functions.